apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -2777,26 +2828,36 @@ private void getAndAssert(ExperimentItem expectedExperimentItem, String workspac
                     .ignoringFields(ITEM_IGNORED_FIELDS)
                     .isEqualTo(expectedExperimentItem);
 
-            assertIgnoredFields(actualExperimentItem, expectedExperimentItem);
+            assertIgnoredFields(actualExperimentItem, expectedExperimentItem, false);
         }
     }
 
     private void assertIgnoredFields(
             List<ExperimentItem> actualExperimentItems, List<ExperimentItem> expectedExperimentItems) {
         assertThat(actualExperimentItems).hasSameSizeAs(expectedExperimentItems);
         for (int i = 0; i < actualExperimentItems.size(); i++) {
-            assertIgnoredFields(actualExperimentItems.get(i), expectedExperimentItems.get(i));
+            assertIgnoredFields(actualExperimentItems.get(i), expectedExperimentItems.get(i), true);
         }
     }
 
-    private void assertIgnoredFields(ExperimentItem actualExperimentItem, ExperimentItem expectedExperimentItem) {
-        assertThat(actualExperimentItem.input()).isNull();
-        assertThat(actualExperimentItem.output()).isNull();
-        assertThat(actualExperimentItem.feedbackScores()).isNull();
+    private void assertIgnoredFields(ExperimentItem actualExperimentItem, ExperimentItem expectedExperimentItem, boolean isFullContent) {
```

Comment:
This is not readable. We could keep this method but create other two which set the boolean values like `assertIgnoredFieldsFullContent` and `assertIgnoredFieldsWithoutFeedbacks`